packages/e2e/test/artillery/wallet-restoration/queries.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Query randomized distinct addresses from db associated with users who are staking
+ */
+export const findAddressesWithRewardsHistory = `
```

Comment:
```suggestion
export const findAddressesWithRegisteredStakeKey = `
```
... or, update the query to join the rewards table

Reply:
Updated with ddac4136263343c727ac0b5603e49a35bda614fb

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/528#discussion_r1032184479